### DISCUSSION TITLE
Jekyll page works on localhost, but broken on Github Pages

### DISCUSSION THREAD

**Discussion by scottcodes** [community/community#21565] [Other Feature Feedback, Questions, & Ideas]
<p>Hi everyone!</p>
<p>I’m a total newbie who is trying to make a blog to document my journey in learning to code.</p>
<p>I followed a few tutorials and have made a site using just the standard minima template, it works locally but when I hosted it on Github Pages, it’s a broken mess. :frowning:</p>
<p>Does anyone know where I made a mistake?</p>
<p>Repo - <a href="https://github.com/scottcodes/blog" class="inline-onebox" rel="noopener nofollow ugc">GitHub - scottcodes/blog</a><br>
Page - <a href="https://scottcodes.github.io/blog/" rel="noopener nofollow ugc">https://scottcodes.github.io/blog/</a></p>
<p>Thank you in advance for your help and advice!</p>

**Comment by BaseMax** [comment]
<p>Hello Scott;</p>
<p>Your repository will works perfectly at FREE <a href="http://github.io" rel="noopener nofollow ugc">github.io</a> domain: <a href="https://scottcodes.github.io/blog/" rel="noopener nofollow ugc">https://scottcodes.github.io/blog/</a></p>
<p>But this will not load perfectly at your custom domain: <a href="http://scottcodes.net/" rel="noopener nofollow ugc">http://scottcodes.net/</a></p>
<p>Because all of the external file links are broken.</p>
<p>for example: <code>http://scottcodes.net/blog/assets/main.css</code> it’s broken and 404<br>
Correct link is: <code>http://scottcodes.net/assets/main.css</code></p>
<p>If I find a chance and time, I will check your repo.</p>
<p>Best wishes;</p>

**Comment by BaseMax** [comment]
<p>Please fix and change <code>baseUrl</code> of external links.<br>
So this will work.</p>
<p>For example: <a href="https://github.com/scottcodes/blog/blob/e96552a0b1c7c3c4ef8a2d3aaf8a60b1d0a4b2ec/_config.yml#L22" class="inline-onebox" rel="noopener nofollow ugc">blog/_config.yml at e96552a0b1c7c3c4ef8a2d3aaf8a60b1d0a4b2ec · scottcodes/blog · GitHub</a></p>

**Comment by scottcodes** [comment]
<p>Hi <a class="mention" href="/basemax">@basemax</a>,</p>
<p>Thank you so much for your response! :slight_smile:</p>
<p>What should the baseURL say instead of “/blog” for it to direct to the correct place? Should it be blank? Or should it say “<a href="https://scottcodes.github.io/" rel="noopener nofollow ugc">https://scottcodes.github.io/</a>” without the “/blog”?</p>

**Comment by BaseMax** [comment]
<p>You are welcome.</p>
<p>The default value of the <code>baseurl</code> at <code>_config.yml</code> is empty string <code>""</code>.<br>
source: <a href="https://jekyllrb.com/docs/configuration/default/" class="inline-onebox" rel="noopener nofollow ugc">Default Configuration | Jekyll • Simple, blog-aware, static sites</a></p>
<p>So I guess you should remove <code>/blog</code> from here and other places you put that.</p>

**Comment by scottcodes** [comment]
<p>Thank you so much <a class="mention" href="/basemax">@basemax</a>! That worked! :slight_smile:</p>
